### PR TITLE
feat - added responsive-grid

### PR DESCRIPTION
**Pull request by ashishnnnnn**
Added Responsive Grid Component .

### CODE REVIEW

**Review by ishapatell1 — APPROVED** [review]
Change in the margin is making it look more good. Good to merge.

**Comment by ashishnnnnn** [comment]
> Change in the margin is making it look more good. Good to merge.
Which element margin.. Could you point that

**Review by anshikaa12 — APPROVED** [review]
code looks good to merge

File: responsive-grid/responsive-grid.html [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+        <div class="flex-center-row list-of-components mar-t-1 gap-0-1">
+          <div id="avatar" class="">
+            <a
+              class="fnt-1-2 pad-1-0 gap-1 flex-center-column"
+              href="../avatar/avatar.html"
+              ><i class="fas fa-user-tie"></i>
+              <p>Avatar</p></a
+            >
+          </div>
+          <div id="alert" class="">
```
Comment (tushartiwari7) [inline responsive-grid/responsive-grid.html]: Remove empty classes.

File: responsive-grid/responsive-grid.html [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+            <a
+              class="fnt-1-2 pad-1-0 gap-1 flex-center-column"
+              href="../alert/alert.html#alert"
+            >
+              <i class="fas fa-exclamation-triangle"></i>
+              <p>Alert</p>
+            </a>
+          </div>
+          <div id="button" class="">
```
Comment (tushartiwari7) [inline responsive-grid/responsive-grid.html]: Same

File: responsive-grid/responsive-grid.html [hunk 3]
```diff
@@ -0,0 +1,329 @@
...
+          <div id="card" class="">
+            <a
+              class="fnt-1-2 pad-1-0 gap-1 flex-center-column"
+              href="../card/card.html#card"
+            >
+              <i class="fas fa-address-card"></i>
+              <p>Card</p>
+            </a>
+          </div>
+          <div id="input" class="#">
```
Comment (tushartiwari7) [inline responsive-grid/responsive-grid.html]: Name class something easy to understand maybe input or something else.

**Review by tushartiwari7 — CHANGES_REQUESTED** [review]
Implement the changes i mentioned and you're good to merge.

**Comment by ashishnnnnn** [comment]
Done .. all the changes..

**Pull request merged**